example/lib/main.dart

```diff
@@ -364,6 +365,31 @@ class _MyAppState extends State<MyApp> {
     child: Text('Publish'),
   );
 
+  Widget getRestChannelHistory() => FlatButton(
```

Comment:
For what it's worth, I think this code could benefit from some commentary. If nothing else for each of the three cases in the `if` statement. It's a little bit opaque on a brief scan. I'm finding myself puzzling over why `_restHistory` could be `null`, but also why you use `?`/`??` at the start but a direct comparison to `== null` later.

This is meant to be example code so we cannot have too much commentary. Many developers taking a look will be time poor so need their hands held.